SliderButton/SliderButtonBehaviorDecorator.js

```diff
@@ -0,0 +1,48 @@
+import platform from '@enact/core/platform';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {findDOMNode} from 'react-dom';
+
+// Adds agate-specific SliderButton behaviors
+// * aria-valuetext handling
+const SliderButtonBehaviorDecorator = (Wrapped) => {
+	const useValueText = ({onChange, ...rest}) => {
```

Comment:
Why does this component naming start with `use`? In my opinion, `useValueText ` is the component, not a hook.

```suggestion
const SliderButtonBehaviorDecorator = (Wrapped) => {
	// eslint-disable-next-line no-shadow
	function SliderButtonBehaviorDecorator ({onChange, ...rest}) {
```

By doing it, we don't have to define a displayName for it.